extension/js/common/composer.ts

```diff
@@ -1707,6 +1718,20 @@ export class Composer {
     }
   }
 
+  private async checkEmailAliases() {
+    if (!this.urlParams.isReplyBox) {
+      const addresses = Value.arr.unique((await Settings.fetchAcctAliasesFromGmail(this.urlParams.acctEmail)).concat(this.urlParams.acctEmail));
+      const storedAdresses = (await Store.getAcct(this.urlParams.acctEmail, ['addresses'])).addresses!;
```

Comment:
A bit dangerous. Type of `(await Store.getAcct(this.urlParams.acctEmail, ['addresses'])).addresses` is `string[] | undefined`. You don't know which one it is unless you test it. Maybe you got `string[]` when you were testing it, but what if sometimes you get `undefined`?

A safer way to deal with this is to add `|| []` instead of `!`. This way you know that either you got an array, or you got undefined but will work with an empty array. Then below the undefined would get fixed after it pulls from Gmail.

Reply:
Okay, got it, I've corrected it. 
I did it that way because I thought that there is always should be at least one element.

Reply:
There should be. But who knows, maybe a bug in other code could cause it to be undefined.